SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -449,7 +449,7 @@ void USpatialConnectionManager::FinishConnecting(Worker_ConnectionFuture* Connec
 			if (ConnectionStatusCode == WORKER_CONNECTION_STATUS_CODE_SUCCESS)
 			{
 				const USpatialGDKSettings* Settings = GetDefault<USpatialGDKSettings>();
-				SpatialConnectionManager->WorkerConnection = NewObject<USpatialWorkerConnection>();
+				SpatialConnectionManager->WorkerConnection = NewObject<USpatialWorkerConnection>(WeakSpatialConnectionManager.Get());
```

Comment:
Seems reasonable, but any specific reason for this change?

Reply:
Yeah it's `ServerWorkerEntityCreator` - it needs a `USpatialNetDriver`, and I get it by `Cast<USpatialNetDriver>(GetWorld()->GetNetDriver())`: https://github.com/spatialos/UnrealGDK/pull/2941/files/034a7c457238b899031b62f928e3970b54537639#diff-5b3c2dbbe58a0fcc21c248d9e9b0f05eff33607797094f3a91ad5d70826ba38dR348-R355
I think I've decided to not add an Init method that would pass in a NetDriver, but that's certainly an option.